server/common.ts

```diff
@@ -399,8 +399,11 @@ interface IMailObject {
 export async function sendMailAsync(mail: IMailObject): Promise<void>  {
 	await sendgrid.send(mail);
 }
-export function sanitize(input: string): string {
-	if (typeof input !== "string") {
+export async function sendBatchMailAsync(mail: IMailObject[]): Promise<void>  {
+	await sendgrid.send(mail);
```

Comment:
You can just return the promise here instead of awaiting it.
Also why does this function exist if it's just a rename of the sendgrid method?

Reply:
Done. I'm just copying the previously existing method (but adding a batch mode). Maybe this was done to allow other email methods in the future?